jsonnet/rules.jsonnet

```diff
@@ -91,6 +106,10 @@
             expr: 'sum(etcd_object_counts) BY (instance)',
             record: 'instance:etcd_object_counts:sum',
           },
+          {
+            expr: 'sum((cluster:master_nodes * on(node) group_left kube_node_status_capacity_cpu_cores) or on(node) (cluster:infra_nodes * on(node) group_left kube_node_status_capacity_cpu_cores) or on(node) (cluster:worker_nodes * on(node) group_left kube_node_status_capacity_cpu_cores)) BY (label_node_openshift_io_os_id, label_kubernetes_io_arch, label_node_role_kubernetes_io_master, label_node_role_kubernetes_io_infra, label_node_role_kubernetes_io_worker)',
```

Comment:
Remove worker, that's an irrelevant label.  worker role has no meaning for us.

Reply:
ok